department-of-veterans-affairs/va.gov-cms
# VACMS-10046: updates for link colors to meet WCAG 2.0 AAA contrast requirements.

https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10046

edit:  8/23  added pr template 

## Description

Closes  #10046

## Testing done
Manually tested on local and Tugboat

## Screenshots
![CleanShot 2022-08-25 at 10 43 24](https://user-images.githubusercontent.com/109987005/186733564-64401c34-2aed-41fc-a97b-8775033992d8.png)


## QA steps
Choose 5 pages at random and check 2 - 3 links per page. Links should reflect the design in the screenshot.

This can be done logged in as any user.
